src/gateway/handlers/interactionCreate.ts

```diff
@@ -52,59 +71,99 @@ export const interactionCreate: GatewayEventHandler = async (
     roles: {}
   }
 
-  if (d.data?.resolved !== undefined) {
-    for (const [id, data] of Object.entries(d.data.resolved.users ?? {})) {
-      await gateway.client.users.set(id, data)
+  if ((d.data as any)?.resolved !== undefined) {

```

Comment:
Why `as any`? I think the data type is pretty predictable.

Reply:
fixed